### DISCUSSION TITLE
yiwen-blog-website/docs/native/ios/uikit/scrollview/

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#227] [Announcements]
# yiwen-blog-website/docs/native/ios/uikit/scrollview/

1. 添加UIscrollview到屏幕中

https://hao-yiwen.github.io/yiwen-blog-website/docs/native/ios/uikit/scrollview/

**Comment by Hao-yiwen** [comment]
https://github.com/Hao-yiwen/ios-study/blob/master/ios-study/resources/ThirdSDKStoryboard.storyboard
